bin/martinize2

```diff
@@ -104,7 +104,7 @@ def pdb_to_universal(system, delete_unknown=False, force_field=None,
     Convert a system read from the PDB to a clean canonical atomistic system.
     """
     if force_field is None:
-        force_field = vermouth.forcefield.get_native_force_field('universal')
+        force_field = vermouth.forcefield.get_native_force_field('charmm')
```

Comment:
Should this be charmm36?

Reply:
I don't think we need the version identifier, because all amber family or charmm family pdbs can be mapped to martini without issues. That is independent of the actual version. So charmm27 will work as well as charmm36 or charmm36m. In fact that is what universal already did to some extent. Naming the ff versions will add a huge amount of redundant files, that we don't really need. It would only be important in case we would want to generate AA parameters, but we already dropped this from the scope. So keeping it just charmm will be fine.

Reply:
Ok, agreed. We can always change it at a later stage (together with a smarter file format to reduce some redundancy)